sites/bizbash/server/styles/index.scss

```diff
@@ -246,6 +255,20 @@ body {
       }
     }
   }
+  &--secondary {
+    #{ $self }__link {
+      @include theme-navbar-link-color($theme-site-bizbash-map, light);
```

Comment:
The root theme should handle this

Reply:
This one needs to be there, since it is using light instead of dark, but both tertiary and primary includes can be removed, since they are using dark (which is what the default is). Thanks! Likely stared too long at the same files last night!

Reply:
Oh, and I did update it to use the theme-site-navbar-map, rather than a separate map on bizbash's file.